docs/guides/06-spatial-aggregations.md

```diff
@@ -1,4 +1,4 @@
-## Aggregating data
+## Spatial aggregations
```

Comment:
Not true, `global*` is not a spatial aggregation

Reply:
🔝